app/views/clientsmodal.html

```diff
@@ -1,126 +1,83 @@
-          <label><span>**</span> Assign Existing Roles</label>
-          <br/>
-          <span ng-repeat="role in roles">
-            <input type="checkbox" ng-model="requiredForNgChange" ng-checked="formData.assigned[role.name]" ng-click="toggleAssignedRoles(role.name)">{{role.name}}</input>
-            <br/>
-          </span>
-          <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>Either '<strong>Assign Existing Roles</strong>' or '<strong>Add New Role</strong>' is required!</div>
-        </div>
-        <div class="form-group" style="float: right;" ng-class="{ 'has-error' : ngError.roles }">
-          <label><span>**</span> Add New Role</label>
-          <input class="form-control" type="text" name="newRole" ng-model="formData.newClientRole" ng-blur="createNewRole()" ng-keyup="$event.keyCode == 13 && createNewRole()"/>
-          <div style="color:orange" ng-show="formData.duplicateNewRole"><i class="glyphicon glyphicon-warning-sign"></i>Role/Client already exists</div>
-          <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>Either '<strong>Assign Existing Roles</strong>' or '<strong>Add New Role</strong>' is required!</div>
-        </div>
-      </div>
-
-      <div class="form-group-2columns">
-        <div class="form-group" style="float: left;" ng-class="{ 'has-error' : ngError.password }">
-          <label ng-if="client.passwordHash" style="float: right">
-            <i tooltip-placement="top" uib-tooltip="A password has been set for this client" class="glyphicon glyphicon-lock"></i>
-          </label>
+<div class="modal-body">
+  <div class="form-group has-error-key">
+    <span>*</span> Indicates a required field<br />
+    <span>*</span><span>*</span> Indicates that at least one of the fields is required
+  </div>
 
-          <label><span>**</span> Basic Auth Password</label>
-          <input class="form-control" placeholder="Set/change password..." type="password" ng-model="temp.password">
-          <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>Either '<strong>Certificate</strong>' or '<strong>Password</strong>' is required!</div>
-        </div>
-        <div class="form-group" style="float: right;" ng-class="{ 'has-error' : ngError.passwordConfirm }">
-          <label>Confirm Password</label>
-          <input class="form-control" placeholder="Confirm password..." type="password" ng-model="temp.passwordConfirm">
-          <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{validationPasswordConfirmMsg}}</div>
-        </div>
-      </div>
+  <fieldset>
+    <uib-tabset justified="true">
+      <!-- ************* TAB FOR CLIENT BASIC INFO ************** -->
+      <uib-tab
+        class="{{ngError.clientBasicInfoTab === true ? 'has-error-tab' : ''}}"
+        active="selectedTab.basicInfo"
+      >
+        <uib-tab-heading
+          ><i
```

Comment:
I think this closing bracket should be on the same line as the tag above. There arent any attributes supplied to that element and doesnt need to be split on a new line